apps/web/src/pages/index.tsx

```diff
@@ -1,23 +1,71 @@
 import { Layout } from "../helper/layout";
 import type { ReactElement } from "react";
+import Image from "next/image";
+import Link from "next/link";
 
 export default function Landing() {
   return (
-    <>
+    <div className="flex flex-col h-[calc(100vh-6rem)] justify-between content-start gap-9">
       <div>
```

Comment:
In the design, this should be a `flex-col` with 36px gaps.

Reply:
gap-9 is exactly 36px so idk what do I need to fix here

Reply:
It's on the wrong `div` I believe. You should inspect and check if gaps actually exist. If they do, apologies.

Reply:
<img width="662" alt="image" src="https://user-images.githubusercontent.com/53175779/193859192-a3685229-3603-4281-9889-b57050309182.png">
they definitely do

Reply:
I meant these gaps. Although I commented this because I noticed that visually there was no gaps in the first review, but I now see that you used other means to accomplish a similar look. If you don't care about getting it pixel-perfect then I guess you can ignore this.

![image](https://user-images.githubusercontent.com/26425747/194109898-3ecbedfc-82d9-4eab-8cc1-cddf6bab6214.png)